README.md

```diff
@@ -83,27 +82,79 @@ The core set of binding data is:
 * **password** - the password or token used to log into the service.  Can be omitted if no authorization required, or take another format such as an API key.  It is strongly recommended that the corresponding ConfigMap metadata properly describes this key.
 * **certificate** - the certificate used by the client to connect to the service.  Can be omitted if no certificate is required, or simply point to another Secret that holds the client certificate.  
 * **uri** - for convenience, the full URI of the service in the form of `<protocol>://<host>:<port>/<name>`.
+* **role-needed** - the name of the role needed to fetch the Secret containing the binding data.  In this scenario a k8s Service Account with the appropriate role must be passed into the binding request (see the [RBAC](#rbac) section below).
 
 Extra binding properties can also be defined (with corresponding metadata) in the bindable service's ConfigMap (or Secret).  For example, services may have credentials that are the same for any user (global setting) in addition to per-user credentials.
 
 
 ### Request service binding
 
-* How do we request a binding from a service (assume the service has been provisioned)
-  * One option:
-    * apiVersion: apps.openshift.io/v1alpha1
-    * kind: ServiceBindingRequest
+Binding is requested by the consuming application, or an entity on its behalf such as the [Runtime Component Operator](https://github.com/application-stacks/runtime-component-operator), via a custom resource that is applied in the same cluster where an implementation of this specification resides.
+
+Since the reference implementation for most of this specification is the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator) we will be using the `ServiceBindingRequest` CRD, which resides in [this folder](https://github.com/redhat-developer/service-binding-operator/tree/master/deploy/crds).  
+
+**Temporary Note**
```

Comment:
This one is also needed to get info out of Kafka CRs: https://github.com/redhat-developer/service-binding-operator/issues/361

Reply:
I agree that's an important one to get solved.  From a spec perspective we need to solve the exact same issue, in the `Pointer to binding data` section (item 1).  Added.